### PR TITLE
change Select to select (Issue 411)

### PR DESCRIPTION
**Pull request by ceb3745**
created this to see the checks run more transparently.

### CODE REVIEW

**Comment by ceb3745** [comment]
@michaelmior This branch looks good to go to resolve the bundlesize issues, unsure why the coverage decreased.

File: src/MultiTable.test.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ it('can change tables browser', () => {
   let browserSelect = wrapper.find('.browserSelect');
 
   // Change the table
-  browserSelect.simulate('change', {value: 'Patient'});
+  browserSelect.simulate('change', {target: {value: 'Patient'}, test: true});
```
Comment (michaelmior) [inline src/MultiTable.test.js]: Why is the `test` parameter here needed?

**Comment by michaelmior** [comment]
Looks like it works! I made a couple comments. Also, a little bit of styling would be nice. I think something like this works well for desktop.

```css
.browserSelect {
  width: 100%;
  font-size: 1em;
  padding: 0.5em;
  margin: 0.5em 0 1em 0;
}
```

File: src/MultiTable.test.js (hunk 1 above)
Comment (ceb3745) [reply src/MultiTable.test.js]: It was an artifact from a previous way I was doing the testing. Updated

**Comment by ceb3745** [comment]
@michaelmior this looks all good to go

**Pull request closed without merging**